optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -114,7 +114,7 @@
         not work on Linux and Mac.</para>
       </note>
 
-      <para>To lower verbosity, the common parts of multiple <literal>&lt;solverBenchmark&gt;</literal> elements are
+      <para xml:id="inheritedSolverBenchmark">To lower verbosity, the common parts of multiple <literal>&lt;solverBenchmark&gt;</literal> elements are
```

Comment:
We don't id's to para's because it's hard select those in the browser when copy pasting into a stackoverflow answer.

It seems as inheritedSolverBenchmark warrants it's own section, so it can be linked to easily?